README.md

```diff
@@ -141,10 +141,14 @@ closes all flumeviews.
 
 set to true if `flumedb.close()` has been called
 
-### flumelog
+---
 
-The log is the heart of `flumedb`, it's the cannonical store of data, and all state is stored there.
-The views are generated completely from the data in the log, so they can be blown away and regenerated easily.
+### flumelog api
```

Comment:
added `<hr>` and `flumelog api` so that it's clearer this is the start of a section. The sizes of heading was hard to distinguish. Also this it now a unique heading which can be an anchor.